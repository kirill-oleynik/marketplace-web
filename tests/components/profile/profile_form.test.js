const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const ProfileForm = require('../../../app/components/profile/profile_form').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <ProfileForm
        t={() => {}}
        errors={{}}
        onSubmit={() => {}}
        revalidation={{}}
        onEmailChange={() => {}}
        needPasswordConfirmation={false}
      />
    );

    expect(toJSON(component)).toMatchSnapshot();
  });
});
