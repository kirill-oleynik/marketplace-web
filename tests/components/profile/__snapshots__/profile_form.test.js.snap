// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="mb-20"
  >
    <div
      className="row"
    >
      <div
        className="col-12 col-sm-6"
      >
        <Main Input
          className="mb-10"
          error={undefined}
          label={undefined}
          name="name"
          onChange={[Function]}
          value={undefined}
        />
      </div>
      <div
        className="col-12 col-sm-6"
      >
        <Main Input
          className="mb-10"
          error={undefined}
          label={undefined}
          name="name"
          onChange={[Function]}
          value={undefined}
        />
      </div>
    </div>
    <Main Input
      className="mb-10"
      error={undefined}
      label={undefined}
      name="email"
      onChange={[Function]}
      type="email"
      value={undefined}
    />
    <Main Input
      className="mb-10"
      error={undefined}
      label={undefined}
      name="organization"
      onChange={[Function]}
      value={undefined}
    />
    <Main Input
      className="mb-10"
      error={undefined}
      label={undefined}
      name="job"
      onChange={[Function]}
      value={undefined}
    />
    <Main Input
      className="mb-10"
      error={undefined}
      label={undefined}
      name="phone"
      onChange={[Function]}
      value={undefined}
    />
  </div>
  <button
    className="main-btn main-btn--blue main-btn--lg w-100 mb-20"
    type="submit"
  />
</form>
`;
